test/Dockerfile

```diff
@@ -1,10 +1,32 @@
-FROM python:3.7-slim
+# This dockerfile is for continuous integration of the mycroft-precise repostiory
+
+# Build an environment that can run the Precise wake word spotter.
+FROM python:3.7-slim as precise-build
 ENV TERM linux
 ENV DEBIAN_FRONTEND noninteractive
 RUN apt-get update && apt-get -y install git python3-scipy cython libhdf5-dev python3-h5py portaudio19-dev swig libpulse-dev libatlas-base-dev
-ADD . mycroft-precise
-WORKDIR mycroft-precise
-RUN pip install .
-RUN pip install pytest
-ENV PYTHONPATH /mycroft-precise
+RUN mkdir -p /root/allure /opt/mycroft/mycroft-precise /root/code-quality
+WORKDIR /opt/mycroft
```

Comment:
This workdir doesn't really make sense in this context.

Reply:
I was using it to keep from having to put `/opt/mycroft` in front of every single file reference.  Why do you think it does not make sense?

Reply:
Ok, so there are some other tools installed in /opt/mycroft that's required. It just feels like a random place to put it when building a custom image. The image root is more common in my experience.